features/main/calendar/atoms/day-labels.tsx

```diff
@@ -0,0 +1,50 @@
+import { END_OF_DAY, nameOfDays, START_OF_DAY } from '@/constants/date';
+import { css, cva, cx } from '@/styled-system/css';
+
+export const DayLabels = () => {
+  return (
+    <div
+      className={css({
+        width: 'full',
+        display: 'flex',
+        gap: '3px',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+      })}
```

Comment:
요 부분은 layoutStyles로 따로 빼주는 것도 좋아보여요!!

```suggestion
<div className={layoutStyles}>
...
const layoutStyles = flex({
  width: 'full',
  gap: '3px',
  justifyContent: 'space-between',
  alignItems: 'center',
});